client/src/sockets/helpers/index.js

```diff
@@ -1,16 +1,37 @@
-import { ListEvents } from 'sockets/enums';
+import { CohortEvents, ListEvents } from 'sockets/enums';
 import history from 'common/utils/history';
+import { Routes } from 'common/constants/enums';
+import { routeGenerator } from 'common/utils/helpers';
 
 export const listEventsController = (event, data, dispatch) => {
```

Comment:
Maybe you should consider to keep the redux-like naming and protocol? 🤔 `(action, payload) => dispatch => { ... }`